lib/flickwerk/patchify.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require "active_support/core_ext/string"
+
+module Flickwerk
+  class Patchify
+    def self.call(directory_name)
+      directory_name ||= "decorators"
+      source_dir = "app/#{directory_name}"
+      target_dir = "app/patches"
+      suffix = directory_name.singularize
```

Comment:
We can be quite sure, because since Zeitwerk they need to comply with Zeitwerk's naming convention. I personally haven't seen a lot of modules without either of those prefixes.